<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLCDNumber" name="lcdNumber"/>
    </item>
    <item>
     <widget class="QPushButton" name="datepushButton">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>vegetables</string>
    </property>
    <addaction name="monitoraction"/>
    <addaction name="temandhhumaction"/>
    <addaction name="illuminationaction"/>
    <addaction name="CO2action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="temandhhumaction">
   <property name="text">
    <string>temandhum</string>
   </property>
  </action>
  <action name="illuminationaction">
   <property name="text">
    <string>illumination</string>
   </property>
  </action>
  <action name="CO2action">
   <property name="text">
    <string>CO2</string>
   </property>
  </action>
  <action name="monitoraction">
   <property name="text">
    <string>monitor</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
